### DISCUSSION TITLE
Account banned ip blocked?

### DISCUSSION THREAD

**Discussion by zolon25** [chromiecraft/chromiecraft#2271] [General]
Hello, I have been playing here on ChromieCraft since September and been having a lot of fun. I would continue playing and am in a guild on here too Odyssey Horde side, but I have a big problem with no explanation. My account has been banned and even ip blocked. Why did this happen? How did this happen? No warnings, just completely banned. I don't recall breaking any terms of use or behavioral issues and have never had any warnings from a GM. Is there a way to restore my account or unblock my ip? Has this happened to anyone else?  Please help, thank you.

**Comment by Amrx2** [comment]
Hi Zolon, Construct here. Could you let me know your account name please so I can have a look into this?

**Reply by zolon25** [reply]
Sure Construct, it's zolon25 for the username. I also had a secondary account for mules zolodon24. I also just tried making a new account to see if that works, nope. It's koregemoth for that username. I've attached the message it gives me trying to log in.  
![Banned](https://user-images.githubusercontent.com/93808720/140621252-33809407-9f07-4a52-a1ab-bcc62df50b4c.jpg)

**Comment by Amrx2** [comment]
Hi there, you went out of your way with your low level orc character, leaving them dead on top of Stormwind fountain. Our rules are clear that we do not allow political messages or discussions no matter the angle. That is why you are currently serving a ban on which runs in 3 days, 12 hours from now.

**Reply by zolon25** [reply]
Uh, that was my Bro. who also shares this IP.  No wonder, he must have done something stupid. :/ He did say he never had any warning from a GM about his joke char. He didn't mean any harm to anyone.

**Comment by Krigsulv** [comment]
hi, im new to this server and i havent even played  yet still my account its banned and i dont know why .
my brother has it own account and says the same, we will like to know the reason.
my username is "Krigsulv".

**Reply by Amrx2** [reply]
Hi, we do not have anybody with an account called Krigsulv on Chromiecraft. Are you sure you connected to our realm?

**Reply by Krigsulv** [reply]
thats wierd im sure i created the account withh that name, then what about Hamrknight thats my other account

**Reply by Amrx2** [reply]
Same again, no player account or character with that name.

**Reply by Krigsulv** [reply]
![2nT8QSxQ(1)](https://user-images.githubusercontent.com/94214716/141614777-b0c1c225-2892-4d54-832d-7cbcedb04bd8.jpeg)

then what does this mean, were am i suposed to create the account

**Reply by Amrx2** [reply]
You need to create an account on our website - https://www.chromiecraft.com/wp-login.php?action=register

Just ensure your realmlist and config.wtf is directed to our server (SET REALMLIST LOGON.CHROMIECRAFT.COM)

**Reply by Krigsulv** [reply]
done , still the same it says im banned the site also comfirmed my accounts are veryfied

**Reply by Amrx2** [reply]
Are you able to login to your account on the website? Can you let me know your IP?
